rust/relay/src/main.rs

```diff
@@ -32,10 +32,15 @@ struct Args {
     #[arg(long, env)]
     public_ip6_addr: Option<Ipv6Addr>,
     /// The address of the local interface where we should serve the prometheus metrics.
-    ///
     /// The metrics will be available at `http://<metrics_addr>/metrics`.
     #[arg(long, env)]
     metrics_addr: Option<SocketAddr>,
+    /// The lowest port used for TURN allocations. See https://www.rfc-editor.org/rfc/rfc8656.html#name-allocations
```

Comment:
This comment will show up in `--help`, not sure if you wanted that.

Reply:
Update to make the RFC link a regular comment.